### ISSUE TITLE
Evaluation on MathVista

### ISSUE THREAD

**Issue by scikkk** [pipilurj/G-LLaVA#6]
Hello! Thanks for your great work, which is very insightful.

I'm now trying to reproduce the results in the paper, and I have already succeeded on the Geo.

However, I'm struggling with MathVista and I’m very confused as to **why there is no content related to MathVista in the code repository, which is the main benchmark demonstrated in the paper**. 

Can you show how to implement the evaluation?

**Comment by ning-mz** [comment]
Hi, may I ask what GPU and how many you used to train. I only has one A800 80G, so I used LoRA to finetune and got the performance around 43% on GeoQA.

**Comment by zzli2022** [comment]
Can you share your reproduction results? I attempted to replicate the results on GeoQA using the dataset provided by the author in LlaVa, but the final scores differ significantly from those provided by the author. Nearly 20 points lower.

**Comment by scikkk** [comment]
![image](https://github.com/pipilurj/G-LLaVA/assets/77485593/f8659554-3562-4577-be03-755cfc7bf673)

**Comment by ning-mz** [comment]
> Can you share your reproduction results? I attempted to replicate the results on GeoQA using the dataset provided by the author in LlaVa, but the final scores differ significantly from those provided by the author. Nearly 20 points lower.

Did you use LoRA? I used LoRA firstly and got around 43% accuracy on GeoQA, than I fully finetune the model and got the correct result.

**Comment by zzli2022** [comment]
我也是，我用LoRA微调得到的结果和你类似

2024-01-19 14:37:31"M.Ning" ***@***.***>写道：

Can you share your reproduction results? I attempted to replicate the results on GeoQA using the dataset provided by the author in LlaVa, but the final scores differ significantly from those provided by the author. Nearly 20 points lower.

Did you use LoRA? I used LoRA firstly and got around 43% accuracy on GeoQA, than I fully finetune the model and got the correct result.

**Comment by zzli2022** [comment]
看起来这个任务非常依赖于全量微调

**Comment by yucheng-zyc** [comment]
> Hello! Thanks for your great work, which is very insightful.
> 
> I'm now trying to reproduce the results in the paper, and I have already succeeded on the Geo.
> 
> However, I'm struggling with MathVista and I’m very confused as to **why there is no content related to MathVista in the code repository, which is the main benchmark demonstrated in the paper**.
> 
> Can you show how to implement the evaluation?

Have you reproduce the result in mathvista? i reproduced the result on Geo but the result on MathVista is poor

**Comment by jweihe** [comment]
Hello everyone,

I see that there’s been a lot of discussion around reproducing results on GeoQA and MathVista. I’m curious if any of you have managed to reproduce results on MathVista, as it appears that there isn't any related content in the code repository.

@scikkk , @yucheng-zyc , have you come across any resources or code for MathVista that could assist in the evaluation process? Any insights or shared experiences would be greatly appreciated!

**Comment by jweihe** [comment]
> Hello everyone,
> 
> I see that there’s been a lot of discussion around reproducing results on GeoQA and MathVista. I’m curious if any of you have managed to reproduce results on MathVista, as it appears that there isn't any related content in the code repository.
> 
> @scikkk , @yucheng-zyc , have you come across any resources or code for MathVista that could assist in the evaluation process? Any insights or shared experiences would be greatly appreciated!

Finished it

**Comment by buhtig-lxz** [comment]
> > Hello everyone,
> > I see that there’s been a lot of discussion around reproducing results on GeoQA and MathVista. I’m curious if any of you have managed to reproduce results on MathVista, as it appears that there isn't any related content in the code repository.
> > @scikkk , @yucheng-zyc , have you come across any resources or code for MathVista that could assist in the evaluation process? Any insights or shared experiences would be greatly appreciated!
> 
> Finished it

Have you successfully reproduced the model's results on MathVista Mountain yet?
